IntelliTect.Trivia.Data/Models/AppUser.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.AspNetCore.Identity;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace IntelliTect.Trivia.Data.Models;
+
+[Read(SecurityPermissionLevels.AllowAll)]
+[Edit(Roles.Admin)]
+[Create(SecurityPermissionLevels.DenyAll)]
+[Delete(SecurityPermissionLevels.DenyAll)]
+public class AppUser: IdentityUser
```

Comment:
Question for Andrew: Does the information on Identity User get exposed to the client via coalesce? (e.g. the password hash?)

Is this the best way to hide it?
```c#
    [InternalUse]
    public override string? PasswordHash { get; set; }
```

Reply:
The answer lies in the generated DTOs, and it is "yes".

And yes.